Makefile

```diff
@@ -27,4 +27,4 @@ build: $(GO_SOURCES) VERSION
 	GO111MODULE=on go build -o $(OUTPUT) ./cmd/finalize-bundle
 
 bundle: build
-	@./bin/finalize-bundle && git status --porcelain | (! grep .) || (echo dirty working tree && false)
\ No newline at end of file
+	@./bin/finalize-bundle && cp -r cnab build/ && cd build && duffle build . && cd ..
```

Comment:
don't you want to run finalize-bundle inside the `build` directory?

Reply:
what happens if there is already a build directory with some files from a previous build?

Reply:
> don't you want to run finalize-bundle inside the build directory?
finalize-bundle will put the newly generated files in the build dir. 
 > what happens if there is already a build directory with some files from a previous build?
added a clean target

Reply:
done!